read-json-in-rust/README.md

```diff
@@ -238,9 +239,11 @@ use std::time::Instant;
 
 use serde_json::Value; // (1)
 
+...
+
 fn load_from_file(path: &str) -> Value { // (2)
     let file = File::open(path).expect("a valid path"); // (4)
-    serde_json::from_reader(file).expect("successfully parsed JSON") // (5)
+    return serde_json::from_reader(file).expect("successfully parsed JSON") // (5)
```

Comment:
```suggestion
    // Idiomatic Rust: implicit return // (5)
    serde_json::from_reader(file).expect("successfully parsed JSON") // (5)
    // or unidiomatic explicit return // (5)
    // return serde_json::from_reader(file).expect("successfully parsed JSON"); // (5)
```

Reply:
This part takes a bit time to get used to, Rust uses implicit return of the last statement (row without `;`), but explicit return can also be used.